### PR TITLE
Receive and show pictures

### PR DESCRIPTION
**Pull request by kindermoumoute**
* Handle POST requests on port 8080
* Get the picture and display it with the pixel lib
* Notify the FPS number

How to test:
* Run the program
*  `curl http://localhost:8080/ -F "file=@test.png"`

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	image img
+}
+
+type img struct {
+	sync.RWMutex
+	frame image.Image
+	count int
+}
+
+var message = []string{"right", "left"}
```
Comment (adrien3d) [inline server/server.go:26]: add forward and stop ?

**Review by adrien3d — CHANGES_REQUESTED** [review]

File: server/server.go (hunk 1 above)
Comment (maxencehenneron) [reply server/server.go:26]: plutot forward et backward

Comment (adrien3d) [reply server/server.go:26]: and stop, when the car reaches a stop signal

Comment (kindermoumoute) [reply server/server.go:26]: This is not intended to return the actual command. This is a message example.

**Pull request merged**